src/main/java/step3/domain/LottoTicket.java

```diff
@@ -0,0 +1,65 @@
+package step3.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+
+  public static final int NUMBERS_COUNT = 6;
+
+  private final List<LottoNumber> numbers;
+
+  // 랜덤 티켓
+  public LottoTicket() {
+    this.numbers = generateRandomLottoNumbers();
+  }
+
+  // 번호를 입력 받는 티켓
+  public LottoTicket(List<LottoNumber> numbers) {
+    this.numbers = numbers;
+    validate();
+  }
+
+  public List<LottoNumber> getNumbers() {
+    return this.numbers;
+  }
+
+  public List<Integer> getNumbersAsInteger() {
+    return this.numbers.stream().map(LottoNumber::getNumber).collect(Collectors.toList());
```

Comment:
```suggestion
    return this.numbers.stream()
                      .map(LottoNumber::getNumber)
                      .collect(Collectors.toList());
```
가독성을 위해 개행을 적절히 섞어보는건 어떨까요?